fineract-provider/gradlew

```diff
@@ -1 +0,0 @@
-../gradlew
```

Comment:
FTR: It took me a sec to understand that you are removing the symlink here.. doesn't work? But probably a good idea anyway, for Windoze consistency.. I noticed that currently on develop we have a fineract-provider/gradlew (symlink), but no fineract-provider/gradlew.bat - that would probably create confusion, so +1 to rm this.